DualCore/HelloWorld.csolution.yml

```diff
@@ -0,0 +1,24 @@
+solution:
+  compiler: AC6
```

Comment:
We should not have a compiler node in csolution if we are steering the compiler selection via .cdefault /command line